save-frontend/src/main/kotlin/com/saveourtool/save/frontend/utils/JsUtils.kt

```diff
@@ -75,3 +109,65 @@ external fun encodeURIComponent(str: String): String
  * @return [Unit]
  */
 fun showGlobalRoleConfirmation() = window.alert(SUPER_ADMIN_MESSAGE)
+
+/**
+ * JS code lines to enable tooltip.
+ *
+ * @return dynamic
+ */
+// language=js
+fun enableTooltip() {
+    js("""
+    var jQuery = require("jquery")
+    require("popper.js")
+    require("bootstrap")
+    jQuery('[data-toggle="tooltip"]').tooltip()
+""")
+}
+
+/**
+ * JS code lines to enable tooltip and popover.
+ *
+ * @return dynamic
+ */
+// language=JS
```

Comment:
Is this comment necessary? If so, can you bring it to the Kdoc?

Reply:
No, as far as I know it is done just as it should be done In order to let Idea highlight the code